scan/views/peers.py

```diff
@@ -106,7 +96,8 @@ def get_context_data(self, **kwargs):
 
         if obj.state == 3 or obj.state == 4: # 4 could be removed if we only want sync
             featured_peers = []
-            for peer in BRS_BOOTSTRAP_PEERS:
+            bootstrap_peers = caching_peers.get_bootstrap_peers()
```

Comment:
> I don't understand what the point is. `caching_peers.get_bootstrap_peers() -> json.loads(os.environ.get("BRS_BOOTSTRAP_PEERS", "[]"))`, but in settings.py `BRS_BOOTSTRAP_PEERS = json.loads(os.environ.get("BRS_BOOTSTRAP_PEERS", "[]"))`

```
def get_bootstrap_peers(self):
        load_dotenv(override=True)
        return json.loads(os.environ.get("BRS_BOOTSTRAP_PEERS", "[]"))
```

load_dotenv loads the most recent BRS_BOOTSTRAP_PEERS. 
At the project startup django core settings (should be)/are immutable and this allows you to update the bootstrap peers list with either additional BRS_BOOTSTRAP_NETWORK or even remove offline on the fly. This could also be done with a model, however that adds extra database calls which is less efficient then simply securely reading/writing the env file.